react/table/public/index.html

```diff
@@ -13,7 +13,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React Table Widget</title>
+    <title>Conferences table</title>
+    <script>
```

Comment:
is this simulating what the login widget should do?

Reply:
the server/backend should append the Keyclock server details for the client